examples/extensions/horizontal-pod-autoscaling/README.md

```diff
@@ -186,13 +175,13 @@ $ while true; do curl http://146.148.6.244; done
 In the case presented here, it increased the number of serving pods to 10:
 
 ```console
-$ kubectl get extensions/hpa
-NAME         REFERENCE                                   TARGET     CURRENT    MINPODS   MAXPODS   AGE
-php-apache   ReplicationController/default/php-apache/   100m cpu   133m cpu   1         10        15m
+$ kubectl get hpa
+NAME         REFERENCE                                   TARGET    CURRENT   MINPODS   MAXPODS   AGE
+php-apache   ReplicationController/default/php-apache/   50%       65%       1         10        14m
```

Comment:
First increase the load current cpu is 305%,  increase the load again, why cpu low to 65%?